README.md

```diff
@@ -1,211 +1,3 @@
 # Acquia Perz CIS Client for PHP
 
-[![Build Status](https://travis-ci.org/acquia/content-hub-php.svg)](https://travis-ci.org/acquia/content-hub-php)
-
 A PHP Client library to consume the Perz CIS API.
-
-## Version Information
```

Comment:
While I'm okay with this for the PR, I added a new ticket to get this readme updated to CIS api endpoint information. https://backlog.acquia.com/browse/DIT-630